app.js

```diff
@@ -1,11 +1,12 @@
 var express = require('express'),
     Habitat = require('habitat'),
     path = require('path'),
+    url = require('url'),
     compression = require('compression'),
     helmet = require('helmet'),
     frameguard = helmet.frameguard,
     reactRouted = require('./dist/lib/react-server-route.js'),
-    getLocale = require('./dist/lib/get-locale.js');
+    locationParser = require('./dist/lib/location-parser.js');
```

Comment:
Just changing the name from getLocale to location parser, as it now doesn't just return a locale, but instead returns an object with locale and a redirect url if redirect is needed. {locale, redirect}